zcash-issue-dag.py

```diff
@@ -92,15 +92,15 @@
 
 REPOS = REPO_SETS[DAG_VIEW]
 
-# Whether to remove issues and PRs that are not target issues.
-ONLY_TARGETS = strtobool(os.environ.get('ONLY_TARGETS', 'false'))
+# Whether to remove issues and PRs that are not target or release issues.
+ONLY_INCLUDE = os.environ.get('ONLY_INCLUDE', '')
```

Comment:
We can only choose one category, not both?

Reply:
Indeed, that was the fastest thing to get working. I wanted you to test and ensure this did what you wanted, before I went further.